cinema/views.py

```diff
@@ -183,7 +234,7 @@
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The permission class `IsAuthenticated` is correctly applied to the `OrderViewSet`, ensuring that only authenticated users can access orders.